apps/tx-builder/src/pages/Dashboard.tsx

```diff
@@ -96,23 +129,23 @@ const Dashboard = (): ReactElement => {
 
           <StyledDivider />
 
-          {/* Address Input */}
+          {/* ABI Address Input */}
           <AddressInput
             id="address"
             name="address"
             label="Enter Address or ENS Name"
             hiddenLabel={false}
-            address={address}
+            address={abiAddress}
```

Comment:
why `abiAddress` instead of `address` ?

Reply:
because we can now have two separate addresses: one for the ABI and another one for implementation